spec/models/order_spec.rb

```diff
@@ -19,6 +19,24 @@
 
         expect(order).to respond_to(:user)
       end
+
+      it 'has many order_items' do
+        order = create(:order)
+
+        expect(order).to respond_to(:order_items)
+      end
+    end
+  end
+
+  describe 'methods' do
+    it 'can calculate total' do
+      order = create(:order)
+      outing = create(:outing)
+      order.order_items.create(order_id: order.id, outing_id: outing.id, quantity: 3)
+      order.order_items.create(order_id: order.id, outing_id: outing.id, quantity: 4)
+
+      expect(order.total).to eq((outing.adjusted_cost * 7))
```

Comment:
@drod1000 I think we should update this test case. Based on the way we create the `cart` object we always group the same `outing.id` within the `cart` therefore we cannot have two `order_item` within the same `order`. Otherwise to test the method functionality this test case is good. FYI @njgheorghita @Dpalazzari